commit 8aed0b01c8372bde746132c6aea3465a12cd164d
Author: Fabio M. Di Nitto <fdinitto@redhat.com>
Date:   Mon Jun 20 09:49:34 2011 +0200

    cman: fix ttl default if no value is specified
    
    when specifing a multicast address, ttl was set to 0 unless
    ttl was explicitly set in cluster.conf.
    
    Resolves: rhbz#713977
    
    Reviewed-by: Lon Hohberger <lhh@redhat.com>
    Signed-off-by: Fabio M. Di Nitto <fdinitto@redhat.com>

diff --git a/cman/daemon/cman-preconfig.c b/cman/daemon/cman-preconfig.c
index 5a73fb3..cde130b 100644
--- a/cman/daemon/cman-preconfig.c
+++ b/cman/daemon/cman-preconfig.c
@@ -691,7 +691,7 @@ static int get_nodename(struct objdb_iface_ver0 *objdb)
 			if (objdb->object_find_next(find_handle2, &mcast_handle) == 0) {
 
 				objdb_get_string(objdb, mcast_handle, "addr", &mcast_name);
-				objdb_get_int(objdb, mcast_handle, "ttl", &ttl, 0);
+				objdb_get_int(objdb, mcast_handle, "ttl", &ttl, ttl);
 			}
 			objdb->object_find_destroy(find_handle2);
 		}
